src/layouts/Facility__c-HEDA Facility Layout.layout

```diff
@@ -82,6 +82,10 @@
         <fields>End_Date__c</fields>
         <relatedList>Course_Offering__c.Facility__c</relatedList>
     </relatedLists>
+    <relatedLists>
```

Comment:
Should there be more fields defined here? I'm actually having trouble figuring out what related list this is - Course Offering Periods, I think?

Reply:
@lifewithryan Not sure if you saw this, forgot to tag you

Reply:
Oh yea, we should figure out what fields to add there...